hw03_karimh.py

```diff
@@ -6,22 +6,122 @@
 # Purpose: To create a code using functions, and submit a pull request.
 # Google Doc Link: https://docs.google.com/document/d/12MLXaEkr6jyzp5jVL2UXoEJTj2VyV12LwCJIk-eW6fs/edit?usp=sharing
 
+
 import turtle
 
-def function_1():
-    "include docstring"
+# Assigning name to the turtle
+Heer = turtle.Turtle()
+
+
+def tyre_1(rad, th, H):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+    "Makes the left tyre of the car"
```

Comment:
Docstrings have a specific format: 

```
'''
Docstring format
'''
```

Reply:
added 3 double-quotations to fix doctring format.